docs/install.rst

```diff
@@ -407,6 +407,7 @@ install sphinx-astropy with conda, graphviz will automatically get installed,
 but if you use pip, you will need to install Graphviz separately as it isn't
 a Python package.
 
+.. _astropy-doc-building:
```

Comment:
This might need a blank line after. I am not 100% sure.

Reply:
I have addressed this in the most recent commit.